core/templates/dev/head/pages/state_editor/StateResponsesDirective.js

```diff
@@ -289,19 +289,32 @@ oppia.directive('stateResponses', [
                 '$scope', '$uibModalInstance', 'ResponsesService',
                 'EditorFirstTimeEventsService', 'StateEditorService',
                 'RuleObjectFactory', 'OutcomeObjectFactory',
-                'COMPONENT_NAME_FEEDBACK', 'GenerateContentIdService',
+                'COMPONENT_NAME_FEEDBACK', 'INTERACTION_SPECS',
+                'GenerateContentIdService', 'StateInteractionIdService',
                 function(
                     $scope, $uibModalInstance, ResponsesService,
                     EditorFirstTimeEventsService, StateEditorService,
                     RuleObjectFactory, OutcomeObjectFactory,
-                    COMPONENT_NAME_FEEDBACK, GenerateContentIdService) {
+                    COMPONENT_NAME_FEEDBACK, INTERACTION_SPECS,
+                    GenerateContentIdService, StateInteractionIdService) {
                   $scope.feedbackEditorIsOpen = false;
                   $scope.addState = addState;
                   $scope.questionModeEnabled =
                     StateEditorService.isInQuestionMode();
                   $scope.openFeedbackEditor = function() {
                     $scope.feedbackEditorIsOpen = true;
                   };
+                  $scope.isCorrectnessFeedbackEnabled = function() {
+                    return StateEditorService.getCorrectnessFeedbackEnabled();
+                  };
+                  $scope.getCurrentInteractionId = function() {
```

Comment:
Do we need this in the scope?

Reply:
No, removed this, used an already existing `getInteractionId()` function.